packages/app/src/components/Layout/YearOverYearLayout/YearOverYearAxis.js

```diff
@@ -16,7 +19,7 @@ const YearOverYearAxis = props => (
         style={{ ...defaultAxisStyles.axisContainer, ...props.style }}
     >
         <div className="label" style={defaultAxisStyles.label}>
-            {getAxisName(props.axisId)}
+            {getAxisNameByLayoutType(props.axisId, LAYOUT_TYPE_YEAR_OVER_YEAR)}
```

Comment:
This may just be my ignorance showing - Does `YEAR_OVER_YEAR` not have a `visType` association?  It seems we should be able to only use `getAxisNameByVisType` (and name it just `getAxisName` to prevent a breaking change) if it did have a `visType`

Reply:
Theoretically yes, we can always access the `visType` somehow in the App, however in this case (`YearOverYearAxis.js`) it's not accessible at the moment but could of course be added. Logically though, from `YearOverYearAxis`'s perspective, the axis name should be fetched explicitly for the layout type `LAYOUT_TYPE_YEAR_OVER_YEAR`, not for the "current visualization type".

Reply:
Regarding the breaking change, this was discussed on Slack and @janhenrikoverland insists on this solution, even though it will cause a breaking change subsequently a version bump.